models/database.go

```diff
@@ -71,7 +71,22 @@ var databaseSchema = [][]string{
 			UNIQUE (address, region)
```

Comment:
@DexterHD That's the biggest reason to use NULLs. We want data integrity on the database level (among other reasons, our users are database experts, so they _will_ try to use SQL instead of APIs 😆). And indexes in both MySQL and PostgreSQL handle NULLs as non-comparable values, to unique indexes work the way we want them to work.

I'm not sure that adding NOT NULL only to several columns worth it. Thoughts?

Reply:
Maybe we should use `NULL` fields only in these cases where we need indexes for example? 
In other cases we can use empty fields and have proper comments in schema about what does empty fields mean. How do you think?